### ISSUE TITLE
Some ideas in French from an institute

### ISSUE THREAD

**Issue by schwabdidier** [schwabdidier/GazePlay#289]
Voici des jeux que j'utilise avec les enfants et que je souhaiterais pouvoir faire avec le contacteur visuel.

www.takatamuser.com

Des jeux de reconnaissance de formes et de couleur sous forme de tri, de tri aussi par taille. j'utilise le site "taka t'amuser" mais il faut cliquer déplacer et recliquer actuellement l'enfant déplace la forme avec le contacteur visuel et je fais le clic avec la souris. il y a aussi le jeu "trouver l'intrus".

http://www.logicieleducatif.fr

Dans le site "logiciel educatif.fr" je fais faire le jeu rangement du plus petit au plus grand, repérage spatial( placer l'ours en peluche dessus, dessous la table, à droite ,...) Il y a aussi un tableau à double entrée très simple.

http://papunet.net/games
sur le site "papunet" je fais faire le puzzle, domino, memory mais il ne sont pas jouables avec le contacteur visuel.

juke box
Pour les ados, je cherche à faire un système de juke box pour qu'ils puissent choisir leur musique avec ou sans clip vidéo.

**Comment by YannRobert** [comment]
Would be great to add mouse moving support and click emulation support in 
https://github.com/coylz/TobiiStreamEngineForJava

The one built in Windows 10 is not reactive enough to be usable in a game. 

This would allow to use many existing web-based (JavaScript or flash) games.

**Comment by schwabdidier** [comment]
Do you know http://precisiongazemouse.com @YannRobert  ?

**Comment by YannRobert** [comment]
> Do you know http://precisiongazemouse.com @YannRobert ?

No, just watched the video, looks good, I'm wondering if it can be use without full cognitive capacity.

**Comment by kuhlkrein** [comment]
@schwabdidier , @YannRobert , I just sent a pull request with this kind option (cf configuration menu),
pressing S (for souris) enable or disable the option when it's activated in configuration menu, and pressing space generate a "left click" event (#499)
